examples/include-external-npm-packages-lock-file/_setup.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
In order to be able to run the fixture in tests, we need to tweak some files (especially because they are copied over a temporary directory)